### DISCUSSION TITLE
devtools middleware cannot serialize BigInt

### DISCUSSION THREAD

**Discussion by montanaflynn** [pmndrs/zustand#2004] [Bug report]
Basically if your store uses any `BigInt`'s and you use devtools middleware it causes this error:

```
TypeError: Do not know how to serialize a BigInt
```

Reproduction: https://codesandbox.io/s/pensive-tesla-ltns7v?file=/src/App.tsx

If you remove the devtools it works fine.

**Comment by dai-shi** [comment]
The serialize option should help.
https://github.com/reduxjs/redux-devtools/blob/135a676949a54d621f621c8fb411e7ca1c594128/extension/docs/API/Arguments.md#serialize
